src/main/java/it/pagopa/pn/deliverypush/utils/StateMap.java

```diff
@@ -255,7 +255,41 @@ public StateMap() {
                 //STATE CHANGE
                 .withTimelineGoToState(TimelineElementCategoryInt.NOTIFICATION_CANCELLED, NotificationStatusInt.CANCELLED, SINGLE_RECIPINET)
         ;
-        this.fromState(NotificationStatusInt.CANCELLED);
+        this.fromState(NotificationStatusInt.CANCELLED)
```

Comment:
Credo che la maggior parte possano essere eliminati teoricamente sono poche le cose conoscesse quando la notifica si trova in annullata